plugins/Jobs.cpp

```diff
@@ -566,14 +570,16 @@ void BedrockJobsCommand::process(SQLite& db) {
 
             // Are we creating a new job, or updating an existing job?
             if (updateJobID) {
-                // Update the existing job.
-                if(!db.writeIdempotent("UPDATE jobs SET "
-                                         "repeat   = " + SQ(SToUpper(job["repeat"])) + ", " +
-                                         "data     = JSON_PATCH(data, " + safeData + "), " +
-                                         "priority = " + SQ(priority) + " " +
-                                       "WHERE jobID = " + SQ(updateJobID) + ";"))
-                {
-                    STHROW("502 update query failed");
+                if (!SContains(job, "overwrite") || job["overwrite"] == "true") {
+                    // Update the existing job.
+                    if(!db.writeIdempotent("UPDATE jobs SET "
+                                             "repeat   = " + SQ(SToUpper(job["repeat"])) + ", " +
+                                             "data     = JSON_PATCH(data, " + safeData + "), " +
```

Comment:
hmmm you sure about PATCH? Don't we want to replace the whole data?

Reply:
This is the same query as before, just indented. I agree it probably should not be a `PATCH`, but I don't want to change existing behavior (in case any relies on this)

Reply:
Oh... 🤦  you are right